package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> That would not be backwards-compatible. I'm wondering if you've actually read the semantic versioning spec?

Yes, I use it for my rust and haskell libraries. And I consider added linter rules as backwards-compatible feature additions, so the minor version will be bumped, not the major one.